google/cloud/spanner_dbapi/cursor.py

```diff
@@ -232,13 +243,18 @@ def description(self):
 
     @property
     def rowcount(self):
+        """Returns number of rows in the table.
+
+        :rtype: int
+        :returns: number of rows
+        """
         return self._row_count
 
     @property
     def is_closed(self):
         """The cursor close indicator.
 
-        :rtype: :class:`bool`
+        :rtype: bool
```

Comment:
That is not a meaningful change. In built documentation (e.g. https://googleapis.dev/python/spanner/latest/index.html) types marked with `:class:` tag will be marked on the documentation page as well. So, it's not major thing, but it's better to use `:class:` tags

Reply:
I'm trying to keep the same code style for docstrings in this rep.

Reply:
This repository was written by Go developer mostly, it has weak Python standards conformity at most part